conf/examples/README.md

```diff
@@ -0,0 +1,7 @@
+# Example Configurations for Goblint
```

Comment:
I think it might be better to have this in our actual documentation instead. There's a "Configuring" page, which is practically empty.